packages/manager/apps/dedicated/client/app/ip/failover/translations/Messages_fr_FR.json

```
@@ -0,0 +1,3 @@
+{
+  "ip_failover_text": "Les IP failover sont des adresses IP publiques qui peuvent être transférées dynamiquement d’un service OVHcloud à l’autre. Ces adresses sont disponibles sous forme d’adresses individuelles ou sous forme de blocs d’adresses."
```

i guess better `à un autre` ?

 ```suggestion
  "ip_failover_text": "Les IP failover sont des adresses IP publiques qui peuvent être transférées dynamiquement d’un service OVHcloud à un autre. Ces adresses sont disponibles sous forme d’adresses individuelles ou sous forme de blocs d’adresses."
```